tests/console/dns_srv.pm

```diff
@@ -36,7 +31,13 @@ sub run {
     systemctl 'show -p SubState named.service|grep SubState=running';
 
     # verify dns server responds to anything
-    assert_script_run "host localhost localhost";
+    my $e = script_run "host localhost localhost";
+    if (!$e) {
+        record_soft_failure 'Bug 1064438: "bind" cannot resolve localhost' if get_var('S390_ZKVM');
+        record_info 'Skip the entire test on bridged networks (e.g. Xen, Hyper-V)' if (is_bridged_networking);
+        return if (is_bridged_networking || get_var('S390_ZKVM'));
+        die 'cannot resolv localost';
```

Comment:
ah, you introduced the typo here :) I suggest to be a bit more verbose, e.g. how about

```perl
die 'Command \'host localhost localhost\' failed, cannot resolv localhost';
```